src/modules/playRoom/pages/PlayRoom.scss

```diff
@@ -0,0 +1,44 @@
+$bg-color-result-container: #dfe6e9;
+$bg-color-head-result: #77b3d4;
+$text-color: #ffffff;
+
+.result-container   {
+    margin-top: 30px;
+    width: 100%;
+    height: 190px;
+    background-color: $bg-color-result-container;
+    overflow: hidden;
+    border-radius: 10px;
+    position: relative;
+    .head-result {
+        display: flex;
+        justify-content: start;
+        align-items: center;
+        height: 40px;
+        background-color: $bg-color-head-result;
+        padding: 15px;
+        h3 {
+            margin-top: 10px;
+            font-size: 16px;
+            color: $text-color;
+            font-weight: 600;
+        }
+    }
+    .result-box {
+        position: relative;
+        &-display {
+            display: flex;
+            justify-content:center;
+            align-items: center;
+            .img-result {
+                margin: 40px 10px 0px 10px;
+                width: 75px;
```

Comment:
Will this break on mobile?

Reply:
Consider the fact that the room for displaying results is pretty small, even on desktop. This might look good because we have not plugged in any other component for PlayRoom page.

Conclusion: Fixed width can cause problem on small devices.

Reply:
I was planning to include it on the phone screen, so this width is appropriate